source/extract_combine_patches.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">%%capture
%matplotlib inline
!pip install git+https://github.com/kornia/kornia
</pre><p>add this somewhere -- otherwise until we release the new functionality won't be availabe (possible by the end of the week)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done